content/components/esp32.md

```diff
@@ -43,12 +43,11 @@ suitable for that variant. Both may be specified (for backwards compatibility) b
 - **partitions** (*Optional*, filename): The name of (optionally including the path to) the file containing the
   partitioning scheme to be used. When not specified, partitions are automatically generated based on `flash_size`.
 
-- **framework** (*Optional*): Options for the underlying framework used by ESPHome. See [Arduino framework](#esp32-arduino_framework)
-  and [ESP-IDF framework](#esp32-espidf_framework).
+- **framework** (*Optional*): Options for the underlying framework used by ESPHome. See [Framework](#esp32-framework).
 
-{{< anchor "esp32-arduino_framework" >}}
+{{< anchor "esp32-framework" >}}
 
-## Arduino framework
+## Framework
 
 This is the default framework for ESP32 chips at the moment.
```

Comment:
Should this line be removed since this section heading has changed and is no longer "Arduino framework"?

Reply:
It was super confusing because the default is different based on variant. I tried to clean that up

Reply:
Love it! Thank you both 🍻